### DISCUSSION TITLE
通过API删除文件后，Web上显示已经删除，但是uploads内依旧存在文件

### DISCUSSION THREAD

**Discussion by HalcyonAzure** [lsky-org/lsky-pro#369] [问答] [answered]
在之前 #366 测试的时候，调用关于Delete的API成功在Web的界面里面显示删除了图片，但是在本地uploads文件夹里面翻阅的时候发现原本的文件依旧存在，不太清楚是不是因为Docker容器或者API调用的时候出现了失误的问题，有其他小伙伴测试也会出现一样的问题吗？

这个问题在正常用网页的上传和下载的时候没什么困扰，但是在用PicGo的时候就会出现网页上以为自己删除了文件，结果实际上服务器本地有文件保留，久而久之缓存越来越多的问题，想问问这个是普遍现象还是说我自己部署导致的特例BUG

~~这里贴上一个自己复现的GIF:~~
~~[情景复现]()~~

BUG在新版本被修复

**Comment by 0xxb** [comment, marked as answer]
> 复现成功，正在排查。

修复了，会在下个版本中发布，感谢反馈！[dec4e6](https://github.com/lsky-org/lsky-pro/commit/dec4e6fdb734834055fca53b118dfb2abb979642)
